GetCertificate/markdown/samples/OK_eucovidcert_details_test_en.md

```diff
@@ -1,6 +1,6 @@
 ## Test Certificate
 ***
-**Certification valid for 48 hours from the time of collection**
+**Certification valid for [48/72] hours from the time of sampling**
```

Comment:
Why both validation periods?

Reply:
just to show the two possible validity times in the example